packages/sdk/src/protocols/store.ts

```diff
@@ -0,0 +1,317 @@
+import { sha256 } from "@noble/hashes/sha256";
+import { StoreCore, waku_store } from "@waku/core";
```

Comment:
let's be consistent - either `camel case` or `snake case`

Reply:
`waku_store` type exports have actually been part for quite long, though I agree we should standardize this.

i propose to use camel case, opened an issue here since the scope of this change will be more than just `store`: 
![Screenshot 2024-03-28 at 1 50 15 PM](https://github.com/waku-org/js-waku/assets/35004822/01c9185d-67d2-4cc2-aa38-a9d1edb385d5)